assets/lokomotive-kubernetes/bootkube/conditional.tf

```diff
@@ -5,6 +5,7 @@ resource "local_file" "calico" {
   content = templatefile("${path.module}/resources/charts/calico.yaml", {
```

Comment:
generic Q: How do we know that it is functional? Any e2e test that validates this behavior?

Reply:
What is functional? The `calico-kube-controllers` process? I have tested it manually, I deleted the node from the cluster and then associated `HostEndpoint` objects have been removed as well.

Reply:
So it was `calico-kube-controllers` and not `calico-hostendpoint-controller` ?

Reply:
Yes, at the moment the only benefit I'm aware of is removal of old `HostEndpoint` objects, which as far as I know, it's not done by our `calico-hostendpoint-controller`. In next Calico releases though, we should be able to also use it for creation of `HostEndpoint` objects.

Reply:
Cool then